Rpackage/R/fitAndScore.R

```diff
@@ -0,0 +1,191 @@
+fit_and_score <- structure(function(tree, target.mat, feature.mat, 
+                                    parameters, feature_names=NULL, 
+                                    n_folds = 3, scorer=NULL, loss = "hinge",
+                                    pruning = TRUE){
+  ### total length
+  l <- length(target.mat[, 1])
+  
+  ### train and test index for each fold
+  fold_split_idx <- NULL
+  
+  ### initial row
+  fold_split_idx$train <- seq(from = 1, to = ((l/n_folds)*(n_folds-1)), by = 1)
+  fold_split_idx$test <- seq(from = ((l/n_folds)*(n_folds-1)) + 1, to = l, by = 1)
+  
+  ### middle values
+  for(n in (n_folds-1):2){
```

Comment:
This code is very complicated and does not seem to work according to my tests. We need to discuss this in the meeting today.

Reply:
Splitting the data into folds can be done very simply, see https://stats.stackexchange.com/a/105839/16717

Reply:
done